packages/terra-paginator/tests/wdio/paginator-spec.js

```diff
@@ -0,0 +1,23 @@
+/* global browser, Terra */
```

Comment:
Let's expanded on the tests for this.

I'd like to get tests for the following:
* Paginator no pages view: /tests/paginator/paginator-no-pages
* Progressive paginator view: /tests/paginator/progressive-paginator